client/my-sites/patterns/components/header/index.tsx

```diff
@@ -2,21 +2,25 @@ import Search from 'calypso/components/search';
 
 import './style.scss';
 
-type PatternsHeaderProps = {
-	title: string;
+type Props = {
```

Comment:
Could we keep the original name? That's the convention we've been following until now 🙂.
```suggestion
type PatternsHeaderProps = {
```

Reply:
I'm fine with sticking to that convention, but for the record, I find it superfluous to mirror the component name in the type name, _unless the props type definition is exported_